multivac/gather_job_data.py

```diff
@@ -120,6 +121,19 @@ def gather_data(self):
                 results[job_failure_type] += 1
                 results['total'] += 1
 
+            # Take data from GitHub Workflow runs API
+            run_file_path = f'{self.workflow_runs_dir}/{job["run_id"]}.json'
+            try:
+                with open(run_file_path) as \
+                        run_file:
+                    run = json.load(run_file)
+                    branch = run['head_branch']
+            except FileNotFoundError:
+                print(f"Can't open {run_file_path}, no runs for job {job['id']}")
```

Comment:
In the first message file path goes first, job id goes second. In the second message it's vice versa. Imho, job id is shorter, so it makes sense to put it first.
```suggestion
                print(f"{job['id']}: can't open run data file {run_file_path}")
```

Reply:
fixed

Reply:
fixed